docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,124 @@
+tags:
+  - gazette
+---
+
+Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one is a jam packed edition.
+
+![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)
+
+Welcome to the _new_ and _improved_ Gatsby Gazette. It's been a while. We have a lot to talk about.
+
+This edition features our work on making Gatsby more stable, the Themes Jam, Jason _Blitz_ Lengstorf, highlights from the community and the revamped Community Maintainers meeting.
+
+## Team Highlights
+
+### Snow Leopard Release
+
+The past few weeks has been all about what we're calling the Snow Leopard release (kudos to [Dustin](https://twitter.com/schaudustin) for the name).
+
+First, an aside.
+
+[Mac OS X Snow Leopard](https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard) (version 10.6) was the seventh major release of Mac OS X (now named macOS). Unlike those of previous versions, the goals of Snow Leopard were improved performance, greater efficiency and the reduction of its overall memory footprint. Addition of new end-user features was not a primary consideration: its name signified its goal to be a refinement of the previous OS X version, Leopard.
+
+Similarly, earlier last month, we felt like we needed to take some time to slow down and clean up Gatsby after some of the big features that we've shipped recently ([themes](https://www.gatsbyjs.org/blog/2019-07-03-announcing-stable-release-gatsby-themes/), [per page manifest](https://www.gatsbyjs.org/blog/2019-06-12-performance-improvements-for-large-sites/)). Our goal was to squash bugs and issues that had happened as a result of moving fast and shipping.
+
+We called this the [Snow Leopard](https://github.com/gatsbyjs/gatsby/projects/13) release.
+
+We think this was a good idea because besides being the best React framework there is, we also want to be the most stable one.
+
+Looks like people think we're doing fine. 🙂
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I have an opinionated starter for creating course sites built with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> that I hadn&#39;t updated since January so I full well expected to spend a few hours fixing it after I updated all the deps.<br><br>Nope, just updated the packages and everything worked. Thanks Gatsby team!</p>&mdash; Brian Holt (@holtbt) <a href="https://twitter.com/holtbt/status/1158559782451609600?ref_src=twsrc%5Etfw">August 6, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Jason Blitz Lengstorf
+
+Jason has basically been living his life on livestream since the past month. He's done some incredible streams and has many more exciting ones coming up.
+
+My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462900728) Mikhail Novikov.
```

Comment:
This seems to be the start of the video.

```suggestion
My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462874512) Mikhail Novikov.
```

Reply:
Aha, thank you!

Reply:
Videos on Twitch are short lived (I think 2 weeks). Is there link for youtube video of that?

Reply:
Correct that Twitch VODs are ephemeral. Given Jason's a Twitch affiliate, he actually gets 2 months before it is removed, but I agree it would be better to use the YT link instead

I don't think he's uploaded this one yet on his YT channel (maybe due to the technical issues on Jason's end during the stream)

Reply:
Yeah, not uploaded to YouTube yet. We'll update this once it's up.